guides/common/modules/proc_installing-fapolicyd-on-server.adoc

```diff
@@ -0,0 +1,61 @@
+[id="installing-fapolicyd-on-server_{context}"]
+ifeval::["{context}" == "{project-context}"]
+= Installing fapolicyd on {ProjectServer}
+endif::[]
+ifeval::["{context}" == "{smart-proxy-context}"]
+= Installing fapolicyd on {SmartProxyServer}
+endif::[]
+
+ifeval::["{context}" == "{project-context}"]
+fapolicyd can either be installed together with {ProjectServer} or can be installed on an existing {ProjectServer}.
+endif::[]
+ifeval::["{context}" == "{smart-proxy-context}"]
+fapolicyd can either be installed together with {SmartProxyServer} or can be installed on an existing {SmartProxyServer}.
+endif::[]
+Use the following procedures to install fapolicyd on your {EL} host.
```

Comment:
We should avoid self-referential descriptions like this (https://redhat-documentation.github.io/supplementary-style-guide/#shortdesc). I think it's perfectly fine to delete this line.

Reply:
Yes :+1: